src/cryptonote_core/tx_verification_utils.cpp

```diff
@@ -227,6 +227,7 @@ static bool ver_non_input_consensus_templated(TxForwardIt tx_begin, TxForwardIt
     std::vector<const rct::rctSig*> rvv;
     rvv.reserve(static_cast<size_t>(std::distance(tx_begin, tx_end)));
 
+    const size_t min_tx_version = hf_version <= HF_VERSION_FCMP_PLUS_PLUS ? 1 : 2;
```

Comment:
I think that this should be `<`, not `<=`. The only v1 transactions allowed now are unmixable sweep transactions, and they're few and far apart enough that they don't need a grace period. It would be better to cut them off early than accidentally introduce some niche bug in the window when all three types of transactions are allowed.